pkg/util/cgroups/cgroupv1_cpu.go

```diff
@@ -42,15 +42,15 @@ func (c *cgroupV1) parseCPUController(stats *CPUStats) {
 		reportError(err)
 	}
 
-	if err := parse2ColumnStats(c.fr, c.pathFor("cpu", "cpu.stat"), 0, 1, func(key, value string) error {
-		intVal, err := strconv.ParseUint(value, 10, 64)
+	if err := parse2ColumnStats(c.fr, c.pathFor("cpu", "cpu.stat"), 0, 1, func(key, value []byte) error {
+		intVal, err := strconv.ParseUint(string(value), 10, 64)
 		if err != nil {
-			reportError(newValueError(value, err))
+			reportError(newValueError(string(value), err))
```

Comment:
here we are doing twice the []byte to string conversion. could you add a new line to instance the string before line 46?